lib/fiware-orion-pep.js

```diff
@@ -85,10 +85,7 @@ function setAuthenticationModule(moduleName) {
  * @param {Function} next        Invokes the next middleware in the chain.
```

Comment:
If this PR is the sum of #143 and #142, it should include the CHANGES_NEXT_RELEASE modifications of both them. (see https://github.com/telefonicaid/fiware-orion-pep/pull/143#discussion_r24022844)